apps/src/applab/designElements/dropdown.jsx

```diff
@@ -148,17 +148,14 @@ module.exports = {
   },
 
   onDeserialize: function (element) {
-    // Don't interfere with focus or clicks in design mode, where the dropdown is
-    // already disabled by jQuery.draggable({cancel: false}).
-    if ($('#divApplab').find(element)[0]) {
-      $(element).on('mousedown', function(e) {
-        if (!Applab.isRunning()) {
-          // Disable dropdown menu unless running
-          e.preventDefault();
-          this.blur();
-          window.focus();
-        }
-      });
-    }
```

Comment:
It turns out preventing `mousedown` in design mode is necessary to prevent right-click in IE from interacting with text area/input and dropdown elements. I was originally worried that preventing `mousedown` in Design Mode would cancel the `click` event handled by `onDesignModeVizClick`, but that turns out not to be true (I was confused because canceling `touchstart` does cancel the `click` event on touch devices).